contrib/core-contract-tests/tests/pox-4/pox_Commands.ts

```diff
@@ -63,6 +64,35 @@ export function PoxCommands(
         r.margin,
       )
     ),
+    // DelegateStackStxCommand
+    fc.record({
+      operator: fc.constantFrom(...wallets.values()),
+      // We can pick a constant from operator wallet.wasDelegatedBy: chain and tuple?
+      stacker: fc.constantFrom(...wallets.values()),
+      // Can call the real to find the current pox cycle and find the max?
+      startBurnHt: fc.nat({ max: 1049}),
+      period: fc.integer({ min: 1, max: 12 }),
+      // Can see the wallet's delegated amount and compare to it? This would replace the margin
+      amount: fc.integer({ min: 1, max: 999999999}),
```

Comment:
The `amount` seems to be not used anywhere?

Reply:
I left it there just in the case we will want to generate a random number smaller than the delegated amount, instead of `minimum amount of ustx` to stack multiplied by `margin`.

Reply:
Try to do what you mentioned: replacing the formula we are using (the one with multiplying the margin) and have the `amount` be a generator of number smaller than the user's balance and greater than the minimum stacking number.

(Don't rush applying this for the other commands, let's see how it works for this command we're working on right now.)